toolkits/nature/packages/nature-facet/scss/50-components/facet.scss

```diff
@@ -0,0 +1,171 @@
+.c-facet {
+	background-color: #f3f3f3;
+	padding: spacing(8);
+	position: relative;
+	font-size: 1.6rem;
+	color: black;
+
+	@include media-query('sm') {
+		display: flex;
+		flex-wrap: wrap;
+		padding: spacing(16);
+	}
+
+	@include media-query('md') {
+		flex-wrap: nowrap;
+	}
+}
+
+.c-facet--small {
+	font-size: 1.4rem;
+}
+
+.c-facet__icon {
+	@include u-icon();
+	flex: 0 0 auto;
+}
+
+.c-facet__label {
+	display: block;
+	font-weight: $context--font-weight-bold;
+	white-space: nowrap;
+	margin-right: spacing(24);
+	margin-bottom: spacing(8);
+}
+
+.c-facet__item {
+	margin-bottom: spacing(8);
+
+	@include media-query('sm') {
+		margin-right: spacing(4);
+
+		&:last-of-type {
+			margin-right: spacing(8);
+		}
+	}
+
+	@include media-query('md') {
+		margin-right: spacing(16);
+		margin-bottom: 0;
+		position: relative;
+	}
+}
+
+.c-facet__item legend {
+	width: 100%;
+
+	@include media-query('sm') {
+		width: auto;
+	}
+}
+
+.c-facet__button {
+	align-items: center;
+	color: black;
+	background: white;
+	border: 1px solid black;
+	border-radius: 3px;
+	display: flex;
+	font-weight: $context--font-weight-normal;
+	margin-bottom: 0;
+	padding: spacing(8);
+	white-space: normal;
+	width: 100%;
+
+	@include media-query('sm') {
+		margin-bottom: 0;
+		white-space: nowrap;
+		width: 111px; // hardcoded to fit 4 items in a row
```

Comment:
Do these hardcoded width values assume the consumer of this component has implemented anything in particular? e.g. u-container must be used on pages that have this component?

Reply:
I'm thinking to refactor these buttons to take up the full width of a container to replace the hardcoded width value. Also within a flexbox container as the intention was to have the buttons on one row unless narrow view where it is stacked. Then have the container widths managed outside of this package as the number of buttons could vary depending on the page if we don't want the buttons to be too wide.

Reply:
Sounds good.

So sorry, I only realised after leaving this comment that this is a Nature scoped package and not Global. But if you do remove the hardcoded values and let those be managed outside of the package, as you suggest, then could this become a Global component instead?